scalafix-core/shared/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -96,8 +96,12 @@ case class ExplicitResultTypes(
     def treeSyntax(tree: Tree): String =
       ScalafixScalametaHacks.resetOrigin(tree).syntax
 
-    def isRuleCandidate[D <: Defn](defn: D, mods: Traversable[Mod], body: Term)(
-        implicit ev: Extract[D, Mod]): Boolean = {
+    def isRuleCandidate[D <: Defn](
+        defn: D,
+        nm: Name,
+        mods: Traversable[Mod],
+        body: Term,
+        printLocalTypes: Boolean)(implicit ev: Extract[D, Mod]): Boolean = {
```

Comment:
is `printLocalTypes` required? Seems `config.skipLocalImplicits` is only used as the argument